### DISCUSSION TITLE
Capacitor PushNotifications plugin is not loaded in release apk

### DISCUSSION THREAD

**Discussion by ReutSchremer** [ionic-team/capacitor#3848] [Q&A] [answered]
Hi
I'm developing react app and want to create android APK for release and publish.

The problem is that when I'm creating release-apk some plugins are not loaded (like PushNotifications and Fcm from community). When I'm using the debug apk everything works just fine- but when I'm creating release apk this plugin is not loaded.

I looked up the plugins that are loaded, and the only thing that I find in common is that they are exported from `web-plugins.js` file.

Anyone handled it before? His android app thought it requires web requirements? I'm really lost.
Thanks guys

![WhatsApp Image 2020-11-23 at 18 10 42](https://user-images.githubusercontent.com/43778158/99986108-76c7c200-2db7-11eb-8bbf-e4f071d168cc.jpeg)
In the above image you can see all the plugins that are available in my app for now.

**Comment by jcesarmobile** [comment, marked as answer]
Do you have `minifyEnabled true` for release builds?
That can prevent plugins from working, if so, change it to `minifyEnabled false`

**Reply by ReutSchremer** [reply]
Wow yes, it solved this. Thank you so much!! 👑👑

(Actually we just deployed on other computer and it worked over there with a whole new configuration, but getting the real answer feels good & makes us independent of using certain computer)
